src/dmd/cli.d

```diff
@@ -239,6 +239,9 @@ dmd -cov -unittest myprog.d
             compiling for symbolic debugging instead of $(B $(LIB)).
             If $(I libname) is not supplied, then no default library is linked in.`
         ),
+        Option("debugmixin",
+            "write out arguments to mixins so that the debuuger has source"
```

Comment:
typo `debugger`

Reply:
Write out where? I suggest indicating that it is written to a file.